lib/DidKeyDriver.js

```diff
@@ -0,0 +1,48 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {LDKeyPair} = require('crypto-ld');
+
+class DidKeyDriver {
+  /**
+   * @param {string} did
+   * @returns {Promise<DidDocument>}
+   * @throws Unsupported Fingerprint Type.
+   */
+  async get({did}) {
+    const fingerprint = did.substr('did:key:'.length);
+    const publicKey = LDKeyPair.fromFingerprint({fingerprint});
+
+    return DidKeyDriver.didDoc({did, publicKey});
+  }
+
+  async generate({keyType = 'Ed25519VerificationKey2018'}) {
+    const publicKey = await LDKeyPair.generate({type: keyType});
+    const did = 'did:key:' + publicKey.fingerprint();
+    return DidKeyDriver.didDoc({did, publicKey});
+  }
+
+  static didDoc({did, publicKey}) {
+    const {publicKeyBase58} = publicKey;
```

Comment:
This makes the assumption that we only support ED25519 keys that which uses the `publicKeyBase58` property and not RSA keys which use the `publicKeyPem`. Is it true that we are not supporting RSA keys?

Reply:
We are not supporting RSA keys for `did:key`.